### PR TITLE
Added "rts" and "dtr" configuration parameters.

### PR DESCRIPTION
**Pull request by cmaglie**
This PR adds the `rts` and `cts` configuration parameter that allows to set and reset RTS and CTS lines.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,10 +1,11 @@
 module github.com/arduino/serial-monitor
 
-go 1.17
+go 1.18
```
Comment (per1234) [inline go.mod:3]: I think this outdated part of the readme should also be updated to say the specific version the project is intended to be built with:

https://github.com/arduino/serial-monitor/blob/4d9899604a779c2a6c5184d439a51f0912c4041e/README.md?plain=1#L7

**Review by per1234 — CHANGES_REQUESTED** [review]

**Review by per1234 — CHANGES_REQUESTED** [review]
UPDATE: fixed!

<s>

The port is always configured as `dtr=on,rts=off` after the change made in https://github.com/arduino/serial-monitor/pull/21/commits/37305bb2a8f19e6907df9d268aefa1021260727a

For example, any of the following commands:

```text
$ arduino-cli monitor -p COM43 --config dtr=off,rts=off
```

```text
$ arduino-cli monitor -p COM43 --config dtr=on,rts=on
```

```text
$ arduino-cli monitor -p COM43 --config dtr=off,rts=on
```

Produce these states:

image

I only checked on Windows. Everything works as expected for me with the version from https://github.com/arduino/serial-monitor/pull/21/commits/4d9899604a779c2a6c5184d439a51f0912c4041e

</s>

**Comment by cmaglie** [comment]
Thanks for testing it @per1234, I messed up with the control bits, now I pushed a new tentative fix in 8e68f5e

**Review by per1234 — APPROVED** [review]
Everything is working as expected on Windows and Linux with the build from https://github.com/arduino/serial-monitor/pull/21/commits/16a3ef63975391a16d796c08eba8d15fdf9a5158

**Pull request merged**